packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx

```diff
@@ -131,12 +143,48 @@ export const WebPropertyDetailPage = (): JSX.Element => {
               aria-label="SPA listing"
             >
               <div className="pf-u-w-33 pf-u-mb-lg pf-u-mt-md">
+              <Split hasGutter className="pf-u-mb-md">
+          <SplitItem className="pf-u-w-33">
                 <SearchInput
                   placeholder="Search by name"
                   value={searchTerm}
                   onChange={(value) => setSearchTerm(value?.toLowerCase())}
                   onClear={() => setSearchTerm('')}
                 />
+                </SplitItem>
+          <SplitItem isFilled />
+          <SplitItem isFilled />
+          <SplitItem>
+            <Select
+              variant={SelectVariant.single}
+              aria-label="filter Input"
+              value="Select Enviroment"
+              onToggle={setIsFilterOpen.toggle}
+              onSelect={(e, value) => {
+                if(value === "Select Enviroment")
+                setFilter('' as string)
+                else
+                setFilter(value as string);
+                setIsFilterOpen.off();
+              }}
+              
+              // selections={filtervalue} //To be kept as Select
+              isOpen={isFilterOpen}
+              aria-labelledby="filter"
+             
+            >
+            
+              {
+              webPropertiesKeys.map((x,y) =>
```

Comment:
Remove the unshift at top and just put the component directly here and change the name from x and y to better variable names.

Reply:
The environment names are coming from the API , the select environment is the default and is not coming in the response

Reply:
Yah. So just put it as component here. then do the map of rest of the data coming from api